queries.yaml

```diff
@@ -93,6 +93,7 @@ layers:
       - vectordatasource.transform.normalize_aerialways
       - vectordatasource.transform.normalize_cycleway
       - vectordatasource.transform.add_is_bicycle_related
+      - vectordatasource.transform.merge_networks_from_tags
```

Comment:
In queries.yaml we'll also want to (selectively) drop shields for new walking, bicycle, and bus at earlier zooms:

- https://github.com/tilezen/vector-datasource/blob/master/queries.yaml#L506-L515

Right now `path` isn't included there – and some international and national important bicycle and walking shields will be present on both paths and `minor_road` so this may get tricky.